web/site/js/vis/message.js

```diff
@@ -126,3 +126,52 @@ export class UnicastMessage extends VObject {
         this._leftFlyTime -= dt
     }
 }
+
+export class AckMessage extends VObject {
+
+    constructor(src, mvInfo) {
+        super();
+
+        this.id = nextMessageId;
+        nextMessageId += 1;
+        this.dstPos = new PIXI.Point(src.position.x, src.position.y + 50)
+        this.mvInfo = mvInfo;
+
+        let size = 10;
+        let sprite = new PIXI.Sprite(Resources().WhiteSolidTriangle32.texture);
+        sprite.tint = this.getColor();
+        sprite.scale.set(size / 32, size / 32);
+        sprite.anchor.set(0.5, 0.5);
+        sprite.position = src.position;
+        this._root = this.sprite = sprite;
+        this._leftFlyTime = 0.7
+    }
+
+    isBroadcast() {
+        return false;
+    }
+
+    getColor() {
+        return 0xaee571;
```

Comment:
Use a constant like ACK_COLOR or the color's name?

Reply:
resolved